pkg/kubelet/kubelet.go

```diff
@@ -548,6 +548,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		iptablesDropBit:                         int(kubeCfg.IPTablesDropBit),
 		experimentalHostUserNamespaceDefaulting: utilfeature.DefaultFeatureGate.Enabled(features.ExperimentalHostUserNamespaceDefaultingGate),
 		keepTerminatedPodVolumes:                keepTerminatedPodVolumes,
+		containerized:                           containerized,
```

Comment:
Actually I would like to remove param `containerized` from `NewMainKubelet`. This param is redundant.

`s.Containerized` is already utilized in `cmd/kubelet/app/server.go`. 

/cc @mtaufen This api change is introduced by your #53088.

Reply:
In `cmd/kubelet/app/server.go`, the sequence of function call is builder --> CreateAndInitKubelet --> NewMainKubelet, so the `containerized` is not utilized

Reply:
I mean `containerized` is redundant in `NewMainKubelet`.

https://github.com/kubernetes/kubernetes/blob/160270800f099538d9ba888201737b4dbab7587a/cmd/kubelet/app/server.go#L141-L145

We won't need  `containerized` for `NewMainKubelet` any more.

Reply:
@dixudx Thanks for your clarify, then I think we can remove the parameter and the field for `kubelet` struct